packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -95,6 +100,14 @@ const pushTransaction = async (request: Request<MessageTypes.PushTransaction>) =
     const rawTx = request.payload.startsWith('0x') ? request.payload.slice(2) : request.payload;
     const api = await request.connect();
     const payload = await api.sendRawTransaction(Buffer.from(rawTx, 'hex'));
+    const { blockhash, lastValidBlockHeight } = await api.getLatestBlockhash('finalized');
+    const confirmStrategy: BlockheightBasedTransactionConfirmationStrategy = {
+        blockhash,
+        lastValidBlockHeight,
```

Comment:
Shouldn't these values be based of the `blockhash` included in the transaction? That's why I originally didn't do it because it would be harder to refactor the code to pass in the correct values.

However I suppose it's not a huge deal, as the only drawback is that the user will simply wait longer for the transaction to fail, but still.

Reply:
I think it could be like that because it's working and also [some thread](https://solana.stackexchange.com/questions/1426/how-to-use-sendandconfirmrawtransaction-with-blockheightbasedtransactionconfirma) on Solana StackOverflow is saying the same

Reply:
Ok. If pending transactions ever are implemented then I'd suggest to fix this part as well to include the `blockchash` from the transaction and its corresponding `lastValidBlockHeight`. As I mentioned in the previous comment - the only drawback of the solution in this PR should be longer than necessary wait times for transaction failure.

Reply:
I think the difference in blockhashes might be, that the txn will be considered expired later (due to this blockhash being younger than the one in the txn). But success / failure should return normally